pkg/yurtctl/cmd/convert/convert.go

```diff
@@ -406,14 +408,28 @@ func (co *ConvertOptions) RunConvert() (err error) {
 		return fmt.Errorf("fail to deploy yurtcontrollermanager: %s", err)
 	}
 	// 4. disable node-controller
-	if err = kubeutil.RunServantJobs(co.clientSet, func(nodeName string) (*batchv1.Job, error) {
-		ctx := map[string]string{
-			"node_servant_image": co.NodeServantImage,
-			"pod_manifest_path":  co.PodMainfestPath,
+	ok, _ := enutil.FileExists(co.PodMainfestPath + "/kube-controller-manager.yaml")
```

Comment:
yurtctl can run on any node in the cluster, not necessarily the master node, so it is not universal to judge whether the kube-controller-manager runs as a static pod or a normal pod based on the existence of the kube-controller-manager.yaml in current node file.